run_bot.sh

```
@@ -64,6 +76,22 @@ create_output_dir() {
 
+# Get the current Docker image name
+get_docker_image() {
+    local image_name=${DOCKER_IMAGE_NAME:-meet-teams-bot:latest}
+    echo "$image_name"
+}
+
+# Find available port
+find_available_port() {
+    local start_port=${1:-3000}
+    local port=$start_port
+    while lsof -Pi :$port -sTCP:LISTEN -t >/dev/null 2>&1; do
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)